src/components/search/types.ts

```diff
@@ -0,0 +1,4 @@
+export default interface SearchState {
+    symbols: [],
```

Comment:
This one is also missing a specific type of array.

Reply:
Added to project now